src/containers/Storage/Pdisk/Pdisk.tsx

```diff
@@ -75,9 +74,9 @@ function Pdisk(props: PDiskProps) {
     const preparePdiskData = () => {
         const {AvailableSize, TotalSize, State, PDiskId, NodeId, Path, Realtime, Device} = props;
         const errorColors = [
-            diskProgressColors[colorSeverity.Orange as keyof typeof diskProgressColors],
-            diskProgressColors[colorSeverity.Red as keyof typeof diskProgressColors],
-            diskProgressColors[colorSeverity.Yellow as keyof typeof diskProgressColors],
+            EFlag.Orange,
+            EFlag.Red,
+            EFlag.Yellow,
```

Comment:
these were unnecessary complicated references to color names
the only purpose of this array is to check whether fields contain values that should be treated as 'errors'

There is room to improvement: seems like what considered an 'error' is based on the `DiskStateProgressBar` appearance, thus the component should be the source of truth for determining whether a color is an 'error' color. But this is another refactoring.